docs/apache-airflow-providers-amazon/connections/chime.rst

```diff
@@ -20,7 +20,9 @@
 Amazon Chime Connection
 ==========================
 
-The Chime connection works with calling Chime webhooks to send messages to a chime room.
+`Amazon Chime <https://aws.amazon.com/chime/>`__ connection works with calling
+`Chime Incoming Webhooks <https://docs.aws.amazon.com/chime/latest/ag/webhooks.html>`__ to send messages to a
```

Comment:
This was worded incorrectly, even in the old version, not your changes specifically. But since you're here making changes anyway:
```suggestion
`Amazon Chime <https://aws.amazon.com/chime/>`__ connection works with
`Chime Incoming Webhooks <https://docs.aws.amazon.com/chime/latest/ag/webhooks.html>`__ to send messages to a
```